src/v2/Apps/Conversation/Components/__tests__/Item.jest.tsx

```diff
@@ -0,0 +1,89 @@
+import { mount } from "enzyme"
+import React from "react"
+import { Item } from "../Item"
+import { Conversation_conversation } from "v2/__generated__/Conversation_conversation.graphql"
+
+describe("Item", () => {
+  describe("when inquiry item is an artwork", () => {
+    // @ts-expect-error STRICT_NULL_CHECK
```

Comment:
These STRICT_NULL_CHECKS were added by a codemod when migrating to strict type checking and should be avoided